.github/workflows/install_test.yaml

```diff
@@ -16,6 +16,9 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest]
         python_version: ["3.8", "3.9", "3.10"]
+        exclude: 
+          - os: macos-latest
+            python-version: "3.8"
```

Comment:
Excluding for now due to https://github.com/catboost/catboost/issues/2371 and https://github.com/actions/setup-python/issues/654